docs/Fontend/01. frontend_init.md

```
@@ -0,0 +1,177 @@
+# SearchWeb 프론트엔드 초기 설정 가이드
+
+## ✅ 완료된 작업
+
+### 1. 프로젝트 구조 설정
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)